src/main/java/com/google/cloud/tools/maven/CloudSdkMojo.java

```diff
@@ -88,6 +96,15 @@ public String getCompileTargetVersion() {
     return javaVersion;
   }
 
+  public void downloadCloudSdk() {
```

Comment:
Would it make sense to put these in defaultCloudSdkBuilder?

Reply:
Maybe, main reason they're separate right now is so I can mock them in tests.